worldedit-core/src/main/java/com/fastasyncworldedit/core/internal/simd/SimdSupport.java

```diff
@@ -44,10 +47,23 @@ public static boolean useVectorApi() {
         return switch (mask) {
             case SingleBlockStateMask single -> vectorizedTargetMask(single.getBlockState().getOrdinalChar());
             case InverseSingleBlockStateMask inverse -> vectorizedTargetMaskInverse(inverse.getBlockState().getOrdinalChar());
+            case ExistingBlockMask ignored -> vectorizedTargetMaskNonAir();
+            case InverseMask inverse -> {
+                final VectorizedMask base = vectorizedTargetMask(inverse.inverse());
+                if (base == null) {
+                    yield null;
+                }
+                yield (set, get) -> base.compareVector(set, get).not();
+            }
             default -> null;
         };
     }
 
+    private static VectorizedMask vectorizedTargetMaskNonAir() {
+        // everything > VOID_AIR is not air
+        return (set, get) -> get.compare(VectorOperators.UNSIGNED_GT, BlockTypesCache.ReservedIDs.VOID_AIR);
```

Comment:
This should just be AIR

Reply:
`ExistingBlockMask` covers all blocks that are considered air, including cave_air and void_air. I'm basically copying that behavior here

Reply:
AIr _can_ be void or cave sometimes, but afaik it never is and is always just AIR

Reply:
cave air is used in structures as it seems. void could probably be ignored, but I don't really want to touch that in any way.

Reply:
Mm yeah fair